apps/web-giddh/src/app/models/api-models/recipt.ts

```diff
@@ -46,7 +46,8 @@ export interface ReceiptItem {
 	dueDate: string;
 	isSelected?: boolean;
 	cashInvoice: boolean;
-	accountCurrencySymbol?: string;
+    accountCurrencySymbol?: string;
```

Comment:
run formatter

Reply:
@rohit-walkover it is due to Github diff the file is already formatted